### PR TITLE
Favicons

### PR DESCRIPTION
**Pull request by samuelli**
Fixes #334

### CODE REVIEW

File: client/manifest.json [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
-  "name": "catalog",
-  "short_name": "catalog",
-  "start_url": "/",
-  "background_color": "#6dc38f",
-  "display": "standalone",
-  "theme_color": "#6dc38f"
+    "name": "Web Components",
+    "icons": [
+        {
+            "src": "\/assets\/android-chrome-192x192.png",
```
Comment (dstockwell) [inline client/manifest.json]: why are these called android-chrome?

**Comment by dstockwell** [comment]
Should `client/assets/apple-touch-icon.png` have a transparent background? (it's white currently)

File: client/manifest.json (hunk 1 above)
Comment (samuelli) [inline client/manifest.json]: Only used by android chrome I guess. Default names the tool spit out. Not sure anyone else uses this manifest

Comment (dstockwell) [inline client/manifest.json]: We should rename them :P

**Comment by samuelli** [comment]
@dstockwell: No, its intentional. iOS does not support transparent icons. If there is transparency, it is replaced with black.

**Comment by abdonrd** [comment]
I recommend check the [index.html](https://github.com/PolymerElements/polymer-starter-kit/blob/master/index.html) and [manifest.json](https://github.com/PolymerElements/polymer-starter-kit/blob/master/manifest.json) from Polymer Starter Kit.

Here a discussion about that: [Update to consistent index.html and manifest.json](https://github.com/Polymer/app-drawer-template/pull/13)

**Pull request merged**